Data/RAList.hs

```diff
@@ -263,6 +268,18 @@ instance (Ord a) => Ord (RAList a) where
     xs >= ys        = toList xs >= toList ys
     xs `compare` ys = toList xs `compare` toList ys
 
+instance NFData a => NFData (RAList a) where
```

Comment:
these NF data instances look correct, but I think  for new enough GHC / base / deepseq  ghc generics give use these for free?